docs/debug_api.md

```diff
@@ -0,0 +1,165 @@
+Debug API
+=========
+
+
+The Debug API is a powerful HTTP API to make adjustments to the bridge at runtime, and is typically useful
+when administrating large bridges. Some functionality has moved to the [Admin Room](./admin_room.md)
```

Comment:
I wonder if it'd make sense to have an aggregate page that lists all the admin/debug functionality, so that a user trying to discover their admin powers won't have to switch between two separate book sections.

Reply:
We're trying to move a lot of the Debug API functionality over to admin rooms, because it turns out Matrix is more convenient than HTTP calls. I'm hoping we can instead reduce the API over time.